src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/OrderStatus.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE dbo.OrderStatus

```

Comment:
Just spoken to @sam-roberts-bjss and we both agree that it would be preferable to have fixed status data initialised at the start. E.g. "Submitted", "Unsubmitted". Would you be able to add this in withe PR?

Reply:
Yeah, no problem 👍